packages/documentation-site/patternfly-docs/content/design-guidelines/components/pagination/pagination.md

```diff
@@ -6,65 +6,86 @@ section: components
 
 <div class="ws-docs-content-img">
-![Example of pagination on mobile table view.](./img/mobile-pagination.svg)
+![Example of a indeterminate pagination.](./img/indeterminate-pagination.svg)
 </div>
 
-1. **Items in view:** At this screen size, the top toolbar only shows the item count to indicate items currently in view. It hides all other pagination controls.
+## Placement
 
-2. **Full pager:** Supplies the same pagination options as the desktop full pager, but in the bottom toolbar.
+### Desktop
 
-### Compact pagination
-
-Compact pagination fits more actions into top toolbars with limited space. Choose this compact variation when your top toolbar contains many items or looks overcrowded. As with full pagination, the compact variation condenses to an item count for mobile views.
-
-#### Desktop compact pagination in toolbar
+For desktop views, place a full pagination component above the content it controls,  typically within a toolbar.
 
 <div class="ws-docs-content-img">
-![Example of a compact pagination.](./img/compact-pagination.svg)
+![an example of a desktop pagination component within a toolbar.](./img/toolbar-pagination.svg)
 </div>
 
-1. **Items in view:** Allows the user to select the item count (number of listed items) per page, as seen in full pagination.
+1. **Per-page selector:**  Used to choose the number of items displayed on a single page at once.
 
-2. **Compact pager:** Supplies the user with page-back and page-next controls only.
+2. **Full pagination:** Supplies data navigation controls, including previous page and next page, first page and last page, and a manual option to type in a desired page number.
 
-### Indeterminate pagination
+When used in a toolbar, pagination may leave limited room for other items like filters or bulk selectors. These additional items will automatically move into an [overflow menu](/components/toolbar/design-guidelines#overflow-menu) as needed.
 
-Use indeterminate pagination if it’s difficult to calculate the total number of items or page count upfront. In these cases, you may use “many” instead of the usual page count that would appear.
+For components with long, scrolling content (like a table), you can add a second full pagination component below the content to ensure that users can navigate without having to scroll back to the top.
+
+### Mobile 
+
+Mobile pagination is formatted differently in order to conserve space. On mobile, full pagination will collapse to a pagination summary with an item count. [Learn how to customize overflow menu breakpoints in our toolbar design guidelines](/components/toolbar/design-guidelines).
 
 <div class="ws-docs-content-img">
-![Example of a indeterminate pagination.](./img/indeterminate-pagination.svg)
+![Example of pagination on mobile table view.](./img/mobile-pagination.svg)
```

Comment:
The bottom pagination in this example should have a top box shadow, indicating it's sticky. Here' s a screenshot from one of our demos

<img width="736" height="868" alt="Image" src="https://github.com/user-attachments/assets/c215566f-28ff-4379-8d74-2134f680ba52" />